en/docs/references/concepts/integrated-windows-authentication-overview.md

```diff
@@ -1,116 +1,62 @@
-to authenticate. That makes the authentication more secure from
-man-in-the-middle type of attacks.
-
-The disadvantage is that since the authentication is done using the
-Windows Active Directory it needs both the client and the server to use
-Microsoft Windows NT based operating systems. Also, the clients need to
-be connected to the domain hosted by the server. This can be used only
-within an intranet. Also, IWA may need some configuration on certain
-browsers like Mozilla Firefox.
+### Cons
+- The client should  always be in the same domain as of the server. IWA is functional only within the Intranet.
+- For some web browsers (ex: Mozilla Firefox), IWA requires extra configurations.
 
 ---
 
-## Implement IWA for a Java Web Application
-
-IWA was initially developed by Microsoft as an authentication mechanism
-for their .NET based IIS servers. However, neither Java nor the server
-applications that host Java web applications (like Tomcat) have native
-support for IWA. There are several third party libraries that provide
-the ability to enable the IWA for Java web applications. Here are
-some of those libraries:
-
--   **JCIFS** – JCIFS is an open source library that had been commonly
-    used a few years ago for IWA authentication of Java web applications.
-    However, this library is no longer maintained and it is not
-    recommended to use because of security flaws.
--   **JESPA** – JESPA is a commercial library that can be used to enable
-    IWA in Java web applications.
--   **Tomcat IWA** – This is Tomcat's implementation of IWA. However,
-    this is not fully completed yet.
--   **SPNEGO** – SPNEGO is an open source library. It uses Java
-    Authentication and Authorization Service (JAAS) for 
-    authentication.
--   **WAFFLE** – WAFFLE is also an open source library. It can be
-    configured easily to use with the Java web applications.  
-
-<!--
-!!! info "Related topics"
+## IWA for a Java web apps
+
+IWA was developed by Microsoft as an authentication mechanism for their .NET based IIS servers.
 
-    -   See [Configuring IWA
-        Single-Sign-On](../../../guides/login/configure-iwa-single-sign-on) to configure
-        integrated windows authentication in WSO2 Identity Server.
--->
+However, neither Java nor the server applications that host Java web applications (like Tomcat) have native support for IWA. There are several third-party libraries that provide the ability to enable the IWA for Java web applications. Following are some of those libraries:
+
+- **JCIFS** is an open-source library that had been commonly used for IWA authentication of Java web applications. However, this library is no longer maintained and it is not recommended using due to security flaws.
+- **JESPA** is a commercial library that can be used to enable IWA in Java web applications.
+- **Tomcat IWA** is Tomcat's implementation of IWA. However, this is not fully completed yet.
```

Comment:
`**Tomcat IWA** is Tomcat's implementation of IWA. However, this is not fully completed yet.`

Is this statement  `However, this is not fully completed yet.` still valid?